applicatie/templates/matrix.html

```diff
@@ -52,40 +52,35 @@
 </style>
 
 
-<body>
+<body bgcolor="#1a1aff">
 
-    <script type="text/javascript" src="{{ url_for('static', filename='js/detail.js') }}" />
-    <script type="text/javascript" src="{{ url_for('static', filename='js/muteren.js') }}" />
+    <script type="text/javascript" src="{{ url_for('static', filename='js/detail.js') }}"></script>
+    <script type="text/javascript" src="{{ url_for('static', filename='js/muteren.js') }}"></script>
 
-    <script>
-        data = {{ data | tojson }};
-        meta = {{ meta | tojson }};
-        contact = {{ contact | tojson }};
-        filename = {{ bestandsnaam | tojson }};
+    <script type="text/javascript">
+        var data = {{ data | tojson }};
+        var meta = {{ meta | tojson }};
+        var contact = {{ contact | tojson }};
+        var filename = {{ bestandsnaam | tojson }};
     </script>
 
-    <body bgcolor="#1a1aff">
```

Comment:
Bij voorkeur <body> op oude plaats laten staan, tenzij het anders niet werkt.

Zie https://stackoverflow.com/questions/3531314/should-i-write-script-in-the-body-or-the-head-of-the-html